pandas/_libs/tslibs/dtypes.pyx

```diff
@@ -348,6 +348,19 @@ cdef int64_t periods_per_day(NPY_DATETIMEUNIT reso=NPY_DATETIMEUNIT.NPY_FR_ns) e
     return day_units
 
 
+cdef int64_t periods_per_second(NPY_DATETIMEUNIT reso) except? -1:
```

Comment:
This can't be defined with `periods_per_day` because of timezones?

Reply:
we could share for some cases, but for not for e.g. NPY_FR_h